app/policies/business_policy.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Authorization policies for businesses
+class BusinessPolicy < ApplicationPolicy
+  # Scope defining which businesses a user has access to
+  class Scope < Scope
+    def resolve
+      if user.admin?
+        scope.all
+      else
+        scope.where(user: user).active
+      end
+    end
+  end
+
+  def update?
+    manage?
+  end
+
+  def destroy?
+    manage?
+  end
+
+  private
+
+  def manage?
```

Comment:
This is a common need, so put it in the parent class. (as `protected`)
Also: "manage" does not convey anything about what this is checking/querying.  ('manage' is vague; it also could have different meanings in different contexts)
These behaviors/queries/methods are all about _who has permissions_  so the method should be named `admin_or_owner?`